### ISSUE TITLE
[XF] EventToCommand behavior feature

### ISSUE THREAD

**Issue by joacar** [PrismLibrary/Prism#714]
In PR #712 the `BehaviorBase<T>` is introduced and that made me think of the general EventToCommand behavior that I have been using. It allows to bind a command to an event, e.g. the _ItemTapped_ event on `ListView`.

Is this something that we would like in Prism or is this considered to be to application specific requirement?

Example

``` c#
<ListView>
    <ListView.Behaviors>
        <behaviors:EventToCommandBehavior EventName="Tapped" Command={Binding ItemTappedCommand} />
    </ListView.Behaviors>
</ListView>
```

**Comment by dvorn** [comment]
> Is this something that we would like in Prism or is this considered to be to application specific requirement?

Prism for WPF already has InvokeCommandAction which is almost the same.

**Comment by dansiegel** [comment]
Personally I would say yes it is something that should be adopted by Prism.Forms. ```BehaviorBase<T>``` was introduced specifically because I believe that over time there may be several behaviors the community would like to have included.

**Comment by adrianknight89** [comment]
Would really like to see this along with basic converters. As of now, I'm using event to command off of [EventToCommandBehavior](https://github.com/xamarin/xamarin-forms-samples/tree/master/Behaviors/EventToCommandBehavior).

**Comment by brianlagunas** [comment]
Just an update for everyone.  The reason this is taking so long for me to look at is because I am heads down recording a new Pluralsight course on Prism for Xamarin.Forms.  This, plus my day job, leaves me little time right now.  I'll check this out as soon as I can.  Luckily this is something that you can add to your projects until it is added to Prism.

**Comment by brianlagunas** [comment]
this has been implemented via #888
